pkg/config/exemptions.go

```diff
@@ -14,10 +15,16 @@ func (conf Configuration) IsActionable(ruleID, controllerName string) bool {
 	}
 
 	for _, example := range conf.Exemptions {
+		if example.Namespace != "" && example.Namespace != namespace {
+			fmt.Println("not matches: " + example.Namespace + " -> " + namespace)
```

Comment:
I think we can get rid of this `Println`

Reply:
ah, sure.. fixed :)